1.8/administration/installing/cloud/aws/upgrading.md

```diff
@@ -0,0 +1,61 @@
+- The [VIP features](/docs/1.8/usage/service-discovery/load-balancing-vips/virtual-ip-addresses/), added in DC/OS 1.8, require that ports 32768 - 65535 are open between all agent and master nodes for both TCP and UDP.
+- The DC/OS UI and APIs may be inconsistent or unavailable while masters are being upgraded. Avoid using them until all masters have been upgraded and have rejoined the cluster. You can monitor the health of a master during an upgrade by watching Exhibitor on port 8181.
+- Task history in the Mesos UI will not persist through the upgrade.
+
+
+## Instructions
+
+1. Login to the current leader master of the cluster.
+   1. Using DC/OS CLI: 
+      ```
+      $dcos node ssh --master-proxy --leader
+      ```
+   1. After you are logged in, run the following command. This command creates a new 1.8 directory (/var/lib/dcos/exhibitor/zookeeper) as a symlink to the old (/var/lib/zookeeper): 
+      ```
+      $for node in $(dig +short master.mesos); do ssh -o StrictHostKeyChecking=no $node "sudo mkdir -p /var/lib/dcos/exhibitor && sudo ln -s /var/lib/zookeeper /var/lib/dcos/exhibitor/zookeeper"; done
+      ```
+
+   1. Go to (http://master-node/exhibitor).
+      * Go to config tab , it should have three fields which have (/var/lib/zookeeper). 
+        ![Exhibitor UI](../img/dcos-exhibitor-fields-before.png)
+        ![Exhibitor UI](../img/dcos-exhibitor-fields-before-2.png)
+      * Edit the config and change all three fields that contain (/var/lib/zookeeper/) to (/var/lib/dcos/exhibitor/zookeeper/).
+        ![Exhibitor UI](../img/dcos-exhibitor-fields-after.png)
+        ![Exhibitor UI](../img/dcos-exhibitor-fields-after-2.png)
+      * Commit and perform a rolling restart.
+      * This will take a couple of minutes and during that time the Exhibitor UI will flash, wait for the commit to be performed fully.
+   1. Make sure the cluster is healthy at this point.
+      * Verify you can access the dashboard.
+      * Verify all components are healthy.
+   
+1. Update Cloudformation stacks.
+   1. Generate the new templates following instructions at [Advanced DC/OS Installation Guide][advanced-aws-custom]. 
+   1. See the AWS documentation on updating CloudFormation stacks: [http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-direct.html](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-direct.html). 
+   1. Updating the zen or infra stack will update the others automatically , if not start with masters then agents.
+   
+1. Deleting instances
+   1. Deleting master nodes.
+      * Start by deleting the non leading master. 
+      * Verify updated master nodes has joined and is healthy.
+      * Once all non leading masters have been replaced, delete the leading master and verify its healthy.
+    
+   1. Deleting agent nodes.
+      * Delete any agent node, order does not matter.
+      * Verify a replacement agent node joins and is healthy.
```

Comment:
Add a comment here that the user can watch the agent node count in the DC/OS UI to confirm the replacement agent joins the cluster.